WorkshopToolCommon/GameBase.cs

```diff
@@ -665,8 +665,20 @@ static bool ProcessItemsUpload(WorkshopType type, List<string> paths, ProcessedO
 
                     if (options.Upload)
                     {
-                        if (mod.Publish())
+                        if (mod.Publish()) 
+                        {
+                            if (!options.DryRun && !string.IsNullOrEmpty(options.DiscordWebhookUrl))
+                            {
+                                MySandboxGame.Log.WriteLineAndConsole(string.Format("Discord-Webhook-Url: {0}", options.DiscordWebhookUrl));
+                                DiscordWebhook hook = new DiscordWebhook(type, mod.Title, changelog);
+                                if (hook.Call(options.DiscordWebhookUrl, out string error))
+                                    MySandboxGame.Log.WriteLineAndConsole("Sent payload to discord webhook");
+                                else
+                                    MySandboxGame.Log.WriteLineAndConsole(string.Format("Discord webhook error: {0}", error));
```

Comment:
I'd recommend changing this to `MySandboxGame.Log.WriteLineError`, or, since the publish already happened anyway and can't be reversed, `MySandboxGame.Log.WriteLineWarning`.
Those are extension methods I added which will colorize the message appropriately.